release-notes/opensearch.release-notes-2.0.0.md

```diff
@@ -0,0 +1,56 @@
+## 2022-05-19 Version 2.0.0 Release Notes
```

Comment:
Shouldnt the release notes include all changes since our last 1.3.x line?
i.e 2.0-RC1[1] notes + diff(2.0, RC1)

[1] https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-2.0.0-rc1.md

Reply:
looking at how we did release 1.0.0, you're right! in that case let me merge the rc1 notes as well.